cms/settings/dev.py

```diff
@@ -57,3 +57,19 @@
 except ImportError:
     pass
 # pylint: enable=unused-wildcard-import,useless-suppression
+
+MIGRATION_LINTER_OPTIONS = {
+    "exclude_apps": [
+        "taggit",
+        "wagtailcore",
+        "wagtailembeds",
+        "wagtailimages",
+        "wagtailadmin",
+        "wagtailsearch",
+        "wagtaildocs",
+        "wagtailredirects",
+        "wagtailusers",
+        "core",
+        "images",
```

Comment:
We should disable specific migrations instead of the whole app, as we want to catch any new migrations that have genuine issues.

Reply:
Neha and I completed this =) [ce114c7](https://github.com/ONSdigital/dis-wagtail/pull/43/commits/ce114c7ec9989047e5c39850b411dd9651c8901b)